node/bft/src/helpers/telemetry.rs

```diff
@@ -182,10 +182,18 @@ impl<N: Network> Telemetry<N> {
             .collect();
 
         // Calculate the certificate participation scores for each validator.
+        // This score is based on how many certificates the validator has included in every two rounds.
+        let tracked_rounds: Vec<_> = tracked_certificates.keys().skip_while(|r| *r % 2 == 0).copied().collect();
         let certificate_participation_scores: IndexMap<_, _> = cfg_iter!(validator_certificates)
             .map(|(address, certificate_rounds)| {
-                // Calculate a rough score for the validator based on the number of certificates seen.
-                let score = certificate_rounds.len() as f64 / num_certificate_rounds as f64 * 100.0;
+                // Count the number of round pairs that are included in the certificate rounds.
+                let num_included_round_pairs = cfg_chunks!(tracked_rounds, 2)
+                    .filter(|chunk| chunk.iter().any(|r| certificate_rounds.contains(r)))
+                    .count();
+                // Calculate the number of round pairs.
+                let num_round_pairs = (tracked_rounds.len().saturating_add(1)).saturating_div(2);
+                // Calculate the score based on the number of certificate rounds the validator is a part of.
+                let score = num_included_round_pairs as f64 / num_round_pairs.max(1) as f64 * 100.0;
```

Comment:
Wouldn't this be faster and easier to read?
```suggestion
                let score = (num_included_round_pairs * 100) / (num_round_pairs.max(1) * 100);
```

Reply:
No, the suggested logic would not work because `num_included_round_pairs` and `num_round_pairs` are of type usize.

An example is (2 * 100) / (3 * 100) = 0 rather than the 0.66 we want to represent it by

Reply:
Sorry, that was supposed to say `(num_included_round_pairs * 100) / num_round_pairs.max(1)`, but it probably does not make a huge difference.

Reply:
I'd opt to keep it as is with floating points for more granularity, but i agree it's a bit wordy